Dockerfile

```diff
@@ -1,40 +1,59 @@
-FROM debian:jessie
+FROM python:2-slim
+MAINTAINER Mozilla Telemetry <telemetry-analysis-service@mozilla.org>
 
 ENV PYTHONUNBUFFERED 1
 ENV PYTHONPATH /app/
 
-RUN set -x \
-    && apt-get update \
-    && apt-get install locales -y \
-    && locale-gen en_US.UTF-8
-
 EXPOSE 8000
 
-RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
-
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        build-essential python python-dev python-pip libpq-dev \
-        postgresql-client gettext sqlite3 libffi-dev \
-    && apt-get autoremove -y \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
+# add a non-privileged user for installing and running the application
+RUN mkdir /app && \
+    chown 10001:0 /app && \
```

Comment:
Yeah, I think so, it's implied in [Dockerflow](https://github.com/mozilla-services/Dockerflow) and used that in https://github.com/mozilla-services/Dockerflow/blob/master/Dockerfile#L3-L6 and other Dockerfiles I've seen in the mozilla GitHub org.

That's not to say that It think that's a particular good choice though, since having $HOME and the folder we mount as a volume during development (/app) being the same can lead to all kinds of funny side effects when some command runs that may want to write to $HOME.

What I think we should do (and update Dockerflow to do that), is separate $HOME for the app user from the WORKDIR, e.g. `/opt/src` for the WORKDIR and `/app` as $HOME.